src/task4/task4.js

```diff
@@ -0,0 +1,9 @@
+function countChar(str, char){
```

Comment:
There is also should be **countBs** function.
Function writing pattern please.
AND lets make one importent rule:
"After each new **{** should be a new 1 tab right line (or 4 scapes, whatever)"

Reply:
my bad